source/AndroidXProject.cshtml

```diff
@@ -183,6 +183,10 @@
     {
         <ProjectReference Include="..\androidx.lifecycle.lifecycle-common\androidx.lifecycle.lifecycle-common.csproj" PrivateAssets="none" />
     }
+    @if (@Model.NuGetPackageId == "Xamarin.AndroidX.Lifecycle.Runtime") 
+    {
+        <ProjectReference Include="..\androidx.core.core-runtime\androidx.core.core-runtime.csproj" PrivateAssets="none" />
+    }
```

Comment:
FYI I tested this change, and it didn't seem to fix the issue for me:
```
03-04 13:26:33.147 15625 15625 I MonoDroid: java.lang.ClassNotFoundException: Didn't find class "androidx.arch.core.executor.ArchTaskExecutor" on path: DexPathList[[zip file "/data/app/com.microsoft.net6.helloandroid-NAw9heRnEfqx7GzblMe_wg==/base.apk"],nativeLibraryDirectories=[/data/app/com.microsoft.net6.helloandroid-NAw9heRnEfqx7GzblMe_wg==/lib/x86, /data/app/com.microsoft.net6.helloandroid-NAw9heRnEfqx7GzblMe_wg==/base.apk!/lib/x86, /system/lib]]
```
I could still fix the problem by including both packages:
```xml
<PackageReference Include="Xamarin.Google.Android.Material" Version="1.3.0.1-net6preview02.4525745" />
<!--TODO: Xamarin.AndroidX.Arch.Core.Runtime should be transitive dependency and not needed-->
<PackageReference Include="Xamarin.AndroidX.Arch.Core.Runtime" Version="2.1.0.8-net6preview02.4525745" />
```

Reply:
Did you change something else here? I didn't see that it solved anything for the .NET 6 packages.